gnomad/utils/release.py

```diff
@@ -2,12 +2,7 @@
 
 from typing import Dict, List, Optional
 
-from gnomad.resources.grch38.gnomad import (
-    GROUPS,
-    POPS,
-    SEXES,
-    SUBSETS,
-)
+from gnomad.resources.grch38.gnomad import GROUPS, POPS, SEXES, SUBSETS
```

Comment:
this edit I'm a bit confused by -- the previous formatting seemed like it was from black. maybe something changed in the newest black version?

Reply:
yeah I'm not sure, I just tried running black on it again, but it didn't change it back